README.md

```diff
@@ -116,7 +124,9 @@ body, that says "hello".
 
 Your Answer:
 ```js
-// your code here
+// $("#greeting").on("click", function(){
+//    $(this).append("<p>hello</p>");
+// });
 ```
 
```

Comment:
Close. The question asked you to append a paragraph to the body of the document. What value does `$(this)` have inside of your event listener?